Neos.ContentGraph.DoctrineDbalAdapter/Classes/Domain/Projection/Feature/NodeDisabling.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\ContentGraph\DoctrineDbalAdapter\Domain\Projection\Feature;
+
+use Doctrine\DBAL\Connection;
+use Neos\EventSourcedContentRepository\Domain\Context\NodeAggregate\Event\NodeAggregateWasDisabled;
+
+/**
+ * The NodeDisabling projection feature trait
+ */
+trait NodeDisabling
+{
+    /**
+     * @throws \Throwable
+     */
+    public function whenNodeAggregateWasDisabled(NodeAggregateWasDisabled $event): void
+    {
+        $this->transactional(function () use ($event) {
+            // TODO: still unsure why we need an "INSERT IGNORE" here;
+            // normal "INSERT" can trigger a duplicate key constraint exception
```

Comment:
I've seen those exceptions too.
IMO we should at least catch the `UniqueConstraintViolation` exception and log/ignore it